go/pkg/tree/tree.go

```diff
@@ -46,6 +46,16 @@ type Stats struct {
 	// TODO(olaola): number of FileMetadata cache hits/misses go here.
 }
 
+// ConstructTreeConfig controls the behavior when constructing a tree from the local filesystem.
+type ConstructTreeConfig struct {
```

Comment:
ConstructConfig? Tree is in package name, so redandant.

Reply:
Done